docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -26,6 +28,7 @@ type:: string
 default:: `syn-thanos`
 
 The namespace in which to create the Grafana dashboard ConfigMaps.
+Multiple instances should't share the same dashboard namespace to avoid name collisions.
```

Comment:
This restriction essentially prohibits you from having Grafana dashboards belonging to different Thanos instances in the same Grafana as far as I understand the implementation.

Reply:
Right. I've added a commit to address this.